xlsx/src/export/worksheets.rs

```diff
@@ -247,15 +248,47 @@ pub(crate) fn get_worksheet_xml(
         format!("<cols>{cols}</cols>")
     };
 
+    let tab_selected = if is_sheet_selected {
+        " tabSelected=\"1\""
+    } else {
+        ""
+    };
+
+    let show_grid_lines = if !worksheet.show_grid_lines {
+        " showGridLines=\"0\""
+    } else {
+        ""
+    };
+
+    let mut active_cell = "A1".to_string();
+    let mut sqref = "A1".to_string();
+
+    let views = &worksheet.views;
+    if let Some(view) = views.get(&0) {
+        let range = view.range;
+        let row = view.row;
+        let column = view.column;
+        let column_name = number_to_column(column).unwrap_or("A".to_string());
+        active_cell = format!("{column_name}{row}");
+
+        let column_start = number_to_column(range[1]).unwrap_or("A".to_string());
+        let column_end = number_to_column(range[3]).unwrap_or("A".to_string());
+        if range[0] == range[2] && range[1] == range[3] {
+            sqref = format!("{column_start}{}", range[0]);
+        } else {
+            sqref = format!("{}{}:{}{}", column_start, range[0], column_end, range[2]);
+        }
+    }
+
     format!(
         "{XML_DECLARATION}
 <worksheet \
 xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" \
 xmlns:r=\"http://schemas.openxmlformats.org/officeDocument/2006/relationships\">\
   <dimension ref=\"{dimension}\"/>\
   <sheetViews>\
-    <sheetView workbookViewId=\"0\">\
-        <selection activeCell=\"A1\" sqref=\"A1\"/>\
+    <sheetView workbookViewId=\"0\"{show_grid_lines}{tab_selected}>\
```

Comment:
Hi @nhatcher, this is great but the selected tab is also on the `xl/workbook.xml` file as:

```xml
<bookViews>
<workbookView xWindow="0" yWindow="500" windowWidth="28800" windowHeight="15800" activeTab="8" xr2:uid="{94D94FA5-4B40-4032-9EB1-15E3426A29B1}"/>
</bookViews>
```
I am not sure what the other options are, are they optional?

Reply:
Good catch! Fixed, I didn't add all the other parameters. Also should we use this when importing?